src/response.ts

```diff
@@ -58,20 +66,24 @@ function createResponseComposition(
           'x-powered-by': 'msw',
         }),
       },
-      overrides,
+      options.mockedResponseOverrides,
     )
+    const transformers = [
+      ...(options.defaultTransformers ?? [stringifyJsonBody]),
+      ...userTransformes,
+    ].filter(Boolean)
```

Comment:
We are applying `compose` from right to left, maybe we should change and apply from left to right?

Reply:
In the functional domain compose works right-to-left. Changing this would be a breaking change for the users.

What are the reasons to change the order?

Reply:
> In the functional domain compose works right-to-left
yes right

> What are the reasons to change the order?
Thinking about FP it is not real problem